doc/topics/reactor/index.rst

```diff
@@ -212,6 +212,10 @@ in :ref:`local reactions <reactor-local>`, but as noted above this is not very
 user-friendly. Therefore, the new config schema is recommended if the master
 is running a supported release.
 
+.. note::
+  state ids of reactors for runners and wheel should all be unique. they can
+  overwrite each other when added to the async queue. causing lost reactions
```

Comment:
Remove the period after queue... add a period at the end.

Reply:
done